feathr_project/feathr/typed_key.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+from copy import copy, deepcopy
+
+from typing import List, Optional
+from feathr.dtype import ValueType, FeatureType
+
+
+class TypedKey:
+    """A feature is an individual measurable property or characteristic of an 'Entity'.
```

Comment:
This is not an Entity anymore, please don't introduce new concepts yet so we don't have to rush to design and implement them